deployワークフローの`sam deploy`が失敗する

## 再現方法

1. mainブランチにpushする
2. deployワークフローが`sam deploy --no-confirm-changeset --no-fail-on-empty-changeset`で失敗する

```
Error: The config profile (dbc) could not be found
Error: Process completed with exit code 1.
```

## 期待する動作

- `sam deploy --no-confirm-changeset --no-fail-on-empty-changeset`が正常に終了してデプロイされる

## ToDo

- [x] ローカルで使用している`dbc`プロファイルをdeployワークフローでは使用しないようにする

--- a/samconfig.toml
+++ b/samconfig.toml
@@ -19,7 +19,6 @@ confirm_changeset = true
 resolve_s3 = true
 s3_prefix = "dlsite-browsing-companion-backend"
 region = "ap-northeast-1"
-profile = "dbc"
 disable_rollback = true
 image_repositories = []
 

